### ISSUE TITLE
1byone drahtloser batteriebetriebener Bewegungsmelder

### ISSUE THREAD

**Issue by Ralf9** [RFD-FHEM/RFFHEM#148]
siehe hier:
https://forum.fhem.de/index.php/topic,67481.msg589347.html#msg589347

```
MU;P0=-1148;P1=-5888;P2=322;P3=-756;P4=-96;P6=-404;P7=1039;D=426767026702676767671702026767670202676767023;CP=2;
MU;P0=-437;P1=172;P3=1011;P4=-1147;P5=303;P6=-5828;D=0103450345030303036345450303034545030303450345030301;CP=3;
MU;P0=124;P1=-463;P2=945;P3=-1168;P4=289;P5=-5837;P6=-204;D=0121234341212123412341212121252343412121234341212123412341212121252343412121234341212123412341212121252343412121234341214641234123412121212523434121212343412121234123414;CP=4;
MU;P0=-108;P1=326;P2=-438;P3=1033;P4=-1122;P5=-5830;P6=104;P7=248;D=01232341234123232323534141232323414123232657;CP=3;
```

**Comment by derchrome** [comment]
Hello Ralf, thanks for transferring this issue to github. In the next days I will use my new superhet receiver to measure the signal again (the one I used to measure the signal was a low budget one). To avoid imprecise timings how about connecting the CC1101 to measure the signal?

I am not sure if this helps but I found some webpages describing the protocol of a 1byone doorbell. The protocol maybe the same as for the PIR.
https://www.google.de/amp/s/arduinodiy.wordpress.com/2015/03/02/qh-832ac-door-bell-modding/amp/
http://goughlui.com/2013/12/20/rtl-sdr-433-92mhz-askook-decoding-of-various-devices-with-rtl_433/
